docs/tutorials/jokes.md

```diff
@@ -2523,7 +2522,6 @@ import type {
   ActionArgs,
   LinksFunction,
 } from "@remix-run/node";
-import { json } from "@remix-run/node";
```

Comment:
Be sure to fix code highlighting as well

Reply:
Thanks for the review, updated @MichaelDeBoey